### PR TITLE
about us page updated

### PR DESCRIPTION
**Pull request by khushimarothi**
### Description

made the page responsive and updated it with some new things

### Checklist

- [x] I am making a proper pull request, not spam.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#315 

## Add relevant screenshot or video (if any)
Screenshot (32)
Screenshot (31)
Screenshot (34)
Screenshot (33)

### CODE REVIEW

**Review by wasimreja — CHANGES_REQUESTED** [review]
beautiful work. some changes are required. 
- before perks section, keep everything as it was.
- remove free and customize how you read. instead you can add along with the books for reading, audio books are also available.
- increase the font size of perks.
```css
font-family: "Josefin Sans", sans-serif;
font-weight: 300;
```
- keep font family of benefit card text to comfortaa.
- keep font weight of card title to 600 and increase margin bottom.

**Comment by khushimarothi** [comment]
Will do rest all the changes.. but I feel that animated about card was not looking good. That's why I just changed it

**Comment by wasimreja** [comment]
> Will do rest all the changes.. but I feel that animated about card was not looking good. That's why I just changed it. Rest what ever you say. Just sharing my opinion

okay. remove the floating animation.

**Comment by wasimreja** [comment]
@khushimarothi all right. remove the floating animation.

**Comment by khushimarothi** [comment]
Please check now

File: about.html [hunk 1]
```diff
@@ -72,7 +72,79 @@ <h1 class="title">
       
     </section>
 
+  
+
+
   <div class="container">
+    <div class="row text-center align-items-center justify-content-center rev p-2 mb-3">
+      <p>“Thanks to Book-Finder, there is no reason to buy books anymore.”<span class="s"> - User</span></p>
```
Comment (wasimreja) [inline about.html:80]:
```suggestion
```
remove this.

**Comment by khushimarothi** [comment]
You deleted the comment .. Should I delete that line. ?? if not then please merge don't keep it on waiting for so long

**Review by wasimreja — CHANGES_REQUESTED** [review]
- before perks section, no need to change anything. the illustration is becoming smaller.  
- add this to perks title.
```css
font-family: "Josefin Sans", sans-serif;
font-weight: 300;
```
- remove the last card and keep only 3 cards. keep the sequence as search in seconds, read anytime, anywhere and audio books respectively.

**Pull request merged**